tasks/install.yaml

```
@@ -0,0 +1,24 @@
+---
+- name: Install Redis
+  when: ansible_distribution == 'Ubuntu'
+  apt:
+    name: redis-server
+
+- when:
+    - redis.replication.sentinel
+    - ansible_distribution == 'Ubuntu'
+  block:
+
+    - name: Check for Sentinel Unit
+      stat:
+        path: /lib/systemd/system/redis-sentinel.service
+      register: redis_sentinel_unit
+      changed_when: false
+
+    - name: Mask Sentinel Unit tp prevent Sentinel from starting directly after installation
+      when: not redis_sentinel_unit.stat.exists
+      command: systemctl mask redis-sentinel
```

```
```suggestion
      command: systemctl mask redis-sentinel
      args:
        creates: /lib/systemd/system/redis-sentinel.service
```